src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
+    <Description>Provides DataFactory management capabilities for Microsoft Azure.</Description>
+    <VersionPrefix>4.13.0</VersionPrefix>
```

Comment:
This new nuget will only contains extended features, such gateway HA feature as mentioned above. We will append it to our public blog and channels announcing the feature public preview and GA.